x-pack/plugins/security_solution/public/hosts/pages/navigation/sessions_tab_body.tsx

```diff
@@ -17,16 +18,17 @@ export const SessionsTabBody = React.memo((alertsProps: AlertsComponentQueryProp
     () => (pageFilters != null ? [...hostNameExistsFilter, ...pageFilters] : hostNameExistsFilter),
     [pageFilters]
   );
+  const isEnterprisePlus = useLicense().isEnterprise();
 
-  return (
+  return isEnterprisePlus ? (
     <SessionsView
       entityType="sessions"
       timelineId={TimelineId.hostsPageSessions}
       {...rest}
       pageFilters={hostPageFilters}
       filterQuery={filterQuery}
     />
-  );
+  ) : null;
```

Comment:
why do we need a license check in this component? Shouldn't we just hide the tab if license is not high enough and then no changes are needed in this file?

Reply:
<img width="1673" alt="Screen Shot 2022-09-14 at 2 57 46 PM" src="https://user-images.githubusercontent.com/8703149/190270476-41d866c3-af93-4845-b835-8a0baabee5eb.png">
we want to prevent users from being able to access the Sessions tab via direct URL, by doing license check on that component users won't be able to see the content of Sessions tab even if they typed the URL

Reply:
<img width="1568" alt="Screen Shot 2022-09-14 at 3 02 23 PM" src="https://user-images.githubusercontent.com/8703149/190270917-fe12d810-31b0-4eb0-b09a-b48f16ff92a9.png">